rfc/2/index.md

```diff
@@ -0,0 +1,459 @@
+                  "name": "z",
+                  "type": "space",
+                  "unit": "nanometer"
+                }
+              ]
+            }
+          ],
+
+          "datasets": [
+            {
+              "path": "1",
+              "coordinateTransformations": [
+                {
+                  "type": "scale",
+                  "scale": [1.0, 11.24, 11.24, 28.0],
+                  "input": "/1",
+                  "output": "root"
+                }
+              ]
+            },
+            {
+              "path": "2-2-1",
+              "coordinateTransformations": [
+                {
+                  "type": "scale",
+                  "scale": [1.0, 22.48, 22.48, 28.0],
+                  "input": "/2-2-1",
+                  "output": "root"
+                }
+              ]
+            },
+            {
+              "path": "4-4-1",
+              "coordinateTransformations": [
+                {
+                  "type": "scale",
+                  "scale": [1.0, 44.96, 44.96, 28.0],
+                  "input": "/4-4-1",
+                  "output": "root"
+                }
+              ]
+            },
+            {
+              "path": "8-8-2",
+              "coordinateTransformations": [
+                {
+                  "type": "scale",
+                  "scale": [1.0, 89.92, 89.92, 56.0],
+                  "input": "/8-8-2",
+                  "output": "root"
+                }
+              ]
+            },
+            {
+              "path": "16-16-4",
+              "coordinateTransformations": [
+                {
+                  "type": "scale",
+                  "scale": [1.0, 179.84, 179.84, 112.0],
+                  "input": "/16-16-4",
+                  "output": "root"
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+
+`456.zarr/1/zarr.json`:
+
+```json
+{
+  "zarr_format": 3,
+  "node_type": "array",
+  "shape": [1, 4096, 4096, 1536],
+  "data_type": "uint8",
+  "chunk_grid": {
+    "configuration": { "chunk_shape": [1, 1024, 1024, 1024] },
+    "name": "regular"
+  },
+  "chunk_key_encoding": {
+    "configuration": { "separator": "/" },
+    "name": "default"
+  },
+  "fill_value": 0,
+  "codecs": [
+    {
+      "configuration": {
+        "chunk_shape": [1, 32, 32, 32],
+        "codecs": [
+          { "name": "transpose", "configuration": { "order": [3, 2, 1, 0] } },
+          { "name": "bytes" },
+          {
+            "name": "blosc",
+            "configuration": {
+              "typesize": 1,
+              "cname": "zstd",
+              "clevel": 5,
+              "shuffle": "noshuffle",
+              "blocksize": 0
+            }
+          }
+        ],
+        "index_codecs": [{ "name": "bytes" }, { "name": "crc32c" }],
+        "index_location": "end"
+      },
+      "name": "sharding_indexed"
+    }
+  ],
+  "attributes": {},
+  "dimension_names": ["c", "x", "y", "z"]
```

Comment:
The `"dimension_names"` field is new to Zarr v3. Will this RFC define the relationship between this field and the `"coordinateSystem"` metadata? For example, it may be specified that their values `MUST` be consistent.

Reply:
I think that would be a reasonable requirement. I'll add it to the text.

Reply:
Thanks for the update! (I don't have permission to mark this as resolved.)